core/templates/dev/head/domain/skill/skill-rights-backend-api.service.ts

```diff
@@ -16,92 +16,97 @@
  * @fileoverview Service to change the rights of skills in the backend.
  */
 
-require('domain/utilities/url-interpolation.service.ts');
+export interface ISkillRightCache {
+  [propName: string]: ISkillRightBackendInterface
+}
 
-angular.module('oppia').factory('SkillRightsBackendApiService', [
-  '$http', '$q', 'UrlInterpolationService', 'SKILL_RIGHTS_URL_TEMPLATE',
-  function($http, $q, UrlInterpolationService, SKILL_RIGHTS_URL_TEMPLATE) {
-    // Maps previously loaded skill rights to their IDs.
-    var skillRightsCache = {};
+import { cloneDeep } from 'lodash';
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { ISkillRightBackendInterface } from
```

Comment:
Separate the Imports Lodash then Angular's then Oppia's.
```
import { cloneDeep } from 'lodash';

import { downgradeInjectable } from '@angular/upgrade/static';
import { HttpClient } from '@angular/common/http';
import { Injectable } from '@angular/core';

import { ISkillRightBackendInterface } from
  'domain/skill/SkillRightsObjectFactory.ts';
import { SkillEditorPageConstants } from
  'pages/skill-editor-page/skill-editor-page.constants.ts';
import { UrlInterpolationService } from
  'domain/utilities/url-interpolation.service';
```

Reply:
Fixed